Binary files orig/vapour/src/.Makevars.ucrt.swp and patched/vapour/src/.Makevars.ucrt.swp differ
diff -Nru orig/vapour/src/Makevars.ucrt patched/vapour/src/Makevars.ucrt
--- orig/vapour/src/Makevars.ucrt	1969-12-31 16:00:00.000000000 -0800
+++ patched/vapour/src/Makevars.ucrt	2021-06-29 02:57:12.428180700 -0700
@@ -0,0 +1,29 @@
+TARGET = lib$(subst gcc,,$(COMPILED_BY))$(R_ARCH)
+
+PKG_CPPFLAGS =\
+	-I../inst/include \
+	-DHAVE_PROJ_H
+
+PKG_LIBS = \
+	-lgdal -lgta -lsqlite3 -lmysqlclient -lspatialite -lproj -lgeos_c -lgeos  \
+	-ljson-c -lnetcdf -lpq \
+	-lwebp -lcurl -lidn2 -lunistring -lssh2 -lgcrypt -lgpg-error -lssl \
+	-lhdf5_hl -lhdf5 -lexpat -lfreexl \
+	-lmfhdf -ldf -lportablexdr  -lpcre \
+	-lopenjp2 -ljasper -lpng -ljpeg -ltiff -lwebp -lgeotiff -lgif -lxml2 -llzma \
+	-lodbc32 -lodbccp32 -liconv -lpsapi -lwldap32 -lsecur32 -lgdi32 -lnormaliz \
+	-lbz2 -lzstd -lz -lcrypto -lcrypt32 -lcfitsio -lws2_32
+
+all: clean winlibs
+
+CXX_STD = CXX11
+
+clean:
+	rm -f $(SHLIB) $(OBJECTS)
+
+winlibs:
+	cp -r $(LOCAL_SOFT)/share/gdal ../inst/
+	cp -r $(LOCAL_SOFT)/share/proj ../inst/
+
+
+.PHONY: all winlibs clean
